src/layouts/contentDiffusion.jsx

```diff
@@ -0,0 +1,94 @@
+// @flow
+
+import React, { type Node } from 'react';
+import { Trans } from '@lingui/react';
+import { Link } from 'gatsby';
+import Img from 'gatsby-image';
+
+import { ChevronRight, targetBlank } from '../layouts/utils';
+
+export const ContentHeader = ({ heading }: { heading: string }) => (
```

Comment:
```suggestion
export const ContentHeader = ({ heading }: {| heading: string |}) => (
```
Can you make the types exact in all these components?